### PR TITLE
Ability to click requests

### PR DESCRIPTION
**Pull request by Patrick7321**

### CODE REVIEW

File: keyform/templates/keyform/request.html [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+{% block css %}
+    {{ block.super }}
+    <link rel="stylesheet" href = "{% static 'keyform/css/request.css' %}"/>
+{% endblock %}
+
+{% block content %}
+    <div class="content">
+        <div class="content-header">{% trans 'Update Key Request' %}</div>
+        <div class="info">
+            <strong><i>{% trans 'Info' %}</i></strong>
```
Comment (enki211) [inline keyform/templates/keyform/request.html]: What about using a span and adding the bold and italicized?

File: keyform/urls.py [hunk 2]
```diff
@@ -4,5 +4,6 @@
 
 urlpatterns = [
     url(r'^$', views.HomeView.as_view(), name='home'),
+    url(r'^request/(?P<pk>\d+)$', views.RequestView.as_view(), name='request'),
```
Comment (enki211) [inline keyform/urls.py]: Maybe `edit-request` instead of just request.

File: keyform/views.py [hunk 3]
```diff
@@ -1,16 +1,24 @@
...
+from django.urls import reverse_lazy, reverse
 
 class HomeView(ListView):
     model = Request
     template_name = "keyform/home.html"
 
+class RequestView(UpdateView):
+    model = Request
+    template_name = "keyform/request.html"
+    fields = ['status']
```
Comment (enki211) [inline keyform/views.py]: Do you need to set the fields on both the view and the form?

File: keyform/static/keyform/css/home.css [hunk 4]
```diff
@@ -2,6 +2,10 @@
     padding: 20px;
 }
 
+.object-list a {
```
Comment (enki211) [inline keyform/static/keyform/css/home.css]: `object-list` is a poor class name choice

File: keyform/templates/keyform/home.html [hunk 5]
```diff
@@ -30,34 +30,36 @@ <h2 class="title">{% trans 'Keyform Requests' %}</h2>
             </div>
         </div>
         {% for object in object_list %}
-            <div class="object row">
-                <div class="small-3 large-3 columns text-center">
-                    {{ object.created_timestamp|date:"m/d/Y" }}
+            <a href = "{% url 'request' pk=object.pk %}">
+                <div class="object row">
```
Comment (enki211) [inline keyform/templates/keyform/home.html]: what does the class object do?

File: keyform/templates/keyform/request.html [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+        <div class="row info-row">
+            <form method=POST> {% csrf_token %}
+                <center>
+                {% for field in form %}
+                    {{ field.label }}: {{ field }}
+                {% endfor %}
+                <input type="submit" value="Update"/>
+                </center>
+            </form>
+            <div class="large-6 columns text-left">
```
Comment (enki211) [inline keyform/templates/keyform/request.html]: What is this empty div for?

File: keyform/templates/keyform/request.html [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+            </div>
+        </div>
+        <br>
+        <div class="row info-row">
+            <form method=POST> {% csrf_token %}
+                <center>
+                {% for field in form %}
+                    {{ field.label }}: {{ field }}
+                {% endfor %}
+                <input type="submit" value="Update"/>
```
Comment (enki211) [inline keyform/templates/keyform/request.html]: trans tag

File: keyform/templates/keyform/request.html [hunk 8]
```diff
@@ -0,0 +1,117 @@
...
+                    </div>
+                    {% if not forloop.last %}<hr width="70%">{% endif %}
+                {% endfor %}
+            </div>
+        </div>
+        <br>
+        <div class="row info-row">
+            <form method=POST> {% csrf_token %}
+                <center>
+                {% for field in form %}
```
Comment (enki211) [inline keyform/templates/keyform/request.html]: Incorrect indentation

**Comment by enki211** [comment]
Looks good

File: keyform/templates/keyform/request.html [hunk 9]
```diff
@@ -0,0 +1,119 @@
...
+                            {% if not forloop.last %}<hr width="70%">{% endif %}
+                        {% endfor %}
+                    </div>
+                {% endif %}
+            {% endwith %}
+        </div>
+        <br>
+        <div class="row info-row">
+            <form method=POST> {% csrf_token %}
+                <center>
```
Comment (Ozrei) [inline keyform/templates/keyform/request.html]: This is a deprecated tag, you shouldn't use it anymore.

**Pull request merged**